dm/tests/openapi/client/openapi_task_check

```diff
@@ -413,6 +413,26 @@ def get_task_status_success(task_name, total):
     print("get_task_status_success resp=", data)
     assert data["total"] == int(total)
 
+def get_task_io_counter_success(task_name, total):
+    url = API_ENDPOINT + "/" + task_name + "/status"
+    resp = requests.get(url=url)
+    data = resp.json()
+    assert resp.status_code == 200
+    print("get_task_io_counter_success resp=", data)
+    assert data["total"] == int(total)
+
+    for item in data.get("data", []):
+        unit = item.get("unit")
+        sync_status = item.get("sync_status", {})
+        # Only validate I/O statistics if the unit is "Sync"
+        if unit == "Sync":
```

Comment:
on cloud, there is a `dump` task, we need to make sure it's value is right too

Reply:
This two fields only return in sync stage. Do you mean that add test for validate `dump` and `load` task

Reply:
we need the `dump_io_total_bytes` for `dump` task, cloud will need it.

Reply:
in fact, `dump_io_total_bytes` is uesd for `sync` task for record network usage from upstream to DM-worker...
Should we rename to such as `sourceIOTotalBytes` and `targetIOTotalBytes`
ref:https://github.com/pingcap/tiflow/commit/7898f06a9ef67379a9143db6c04c68cf54e2ef78#diff-4e318b3b5571719c456aef097f799cd5393619ce320c58dc918c89c0ce5d25d0R473

Reply:
dump is here
https://github.com/pingcap/tiflow/blob/98a610a051866cfcc5af1b58bbcada92eb423f8e/dm/dumpling/dumpling.go#L283-L293

load is here
https://github.com/pingcap/tiflow/blob/98a610a051866cfcc5af1b58bbcada92eb423f8e/dm/loader/lightning.go#L635-L642

they all need to be exposed

Reply:
all dumpstatus and loadstatus fields had exposed in openapi, but i only chose few number/stage field to test. 
https://github.com/pingcap/tiflow/pull/11742/files#diff-e9ab88b6967b80315b97f736b2bb8f4fe37e2b35b98c5f27cf667c011209c9ff

And dump task, load task(test by full mode) and sync task(test by all mode) add related tests